src/pybind/mgr/dashboard/frontend/src/app/shared/api/settings.service.ts

```diff
@@ -69,7 +69,7 @@ export class SettingsService {
   }
 
   private getSettingsValue(data: any): string {
-    return data.value || data.instance || '';
+    return data.value || data.instance || data || '';
```

Comment:
?

Reply:
Removed it.
 It was a change to accomodate the other endpoints I added, no longer needed